### PR TITLE
website/integrations: add Chronograf

### PR DESCRIPTION
**Pull request by tomlawesome**
Add Chronograf oauth integration to website.

## Details

Chronograf is not listed yet in integrations, but it's working well for me using their generic oauth and Authentik's oauth endpoints generated by simply creating an oauth provider.

---

## Checklist

-   [ ] Local tests pass (`ak test authentik/`)
-   [ ] The code has been formatted (`make lint-fix`)

If an API change has been made

-   [ ] The API schema has been updated (`make gen-build`)

If changes to the frontend have been made

-   [ ] The code has been formatted (`make web`)
-   [ ] The translation files have been updated (`make i18n-extract`)

If applicable

-   [x] The documentation has been updated
-   [ ] The documentation has been formatted (`make website`)

### CODE REVIEW

**Comment by tanberry** [comment]
Thanks so much @tomlawesome for this contribution! I'll do a copy-edit on the docs, and get a tech review on it.  Always nice to add more integrations.

**Review by rissson — CHANGES_REQUESTED** [review]

File: website/integrations/services/chronograf/index.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+      GENERIC_SCOPES: "email,profile,openid"
+      GENERIC_DOMAINS: "authentik.company"
+      GENERIC_AUTH_URL: "https://auth.authentik.company/application/o/authorize/"
+      GENERIC_TOKEN_URL: "https://auth.authentik.company/application/o/token/"
+      GENERIC_API_URL: "https://auth.authentik.company/application/o/userinfo/"
+      GENERIC_API_KEY: "email"
+```
+In this configuration, "GENERIC_NAME" is what will appear on the Chronograf login page:
+
+![image](https://github.com/tomlawesome/authentik/assets/76453276/c14a4694-563b-4a94-9cd4-162c4e543bd7)
```
[the remark this answers is not in the document]
Comment (tomlawesome) [reply website/integrations/services/chronograf/index.md:47]: I'm not sure, sorry. Very new to pull requests!

[the remark this answers is not in the document]
Comment (rissson) [reply website/integrations/services/chronograf/index.md:47]: Sure, no worries. What you can do is directly put the image under `website/integrations/services/chronograf/login.png` and use `` to show the image.

**Review by rissson — CHANGES_REQUESTED** [review]
I missed it before, but you also need to add this page to `website/sidebarsIntegrations.js`, probably under the `Monitoring` category (the list is alphabetically sorted).

**Comment by PierreBtz** [comment]
Hey, thanks a lot for this PR, this helped a lot.

However I had to add `USE_ID_TOKEN=true` for the setup to properly work (as per https://docs.influxdata.com/chronograf/v1/administration/managing-security/#jwks-signature-verification-optional).

**Comment by tanberry** [comment]
Hi @tomlawesome can you take a look at the last few points/edits, and then see about adding the tip that @PierreBtz made? Also, you'll need to run `make website` or `make website-lint-fix` before pushing your final commit, so that the build check for linting passes.

Apologies we let this one sit for a bot; we are looking forward to your reply and getting it merged, thanks!

**Pull request closed without merging**